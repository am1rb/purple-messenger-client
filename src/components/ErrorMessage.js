// @flow

import React from 'react'

import styles from './ErrorMessage.css'

type Props = {
	message?: string,
	children?: any,
};

export default class ErrorMessage extends React.PureComponent<Props>
{
	render()
	{
		return (
			<div className={styles.error}>
				{this.props.message||this.props.children}
			</div>
		)
	}
}