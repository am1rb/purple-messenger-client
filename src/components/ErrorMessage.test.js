// @flow

import React from 'react'
import renderer from 'react-test-renderer'

import ErrorMessage from './ErrorMessage'


describe("<ErrorMessage /> component tests", () => {

	it("Should not regress", () => {

		const component = renderer.create(
			<ErrorMessage label="This is error message" />
		)

		expect(component.toJSON()).toMatchSnapshot()

	})

})
