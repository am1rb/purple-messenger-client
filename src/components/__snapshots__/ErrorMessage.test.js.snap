// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage /> component tests Should not regress 1`] = `
<div
  className={undefined}
/>
`;
